src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: the first leg is ")
+                .append(firstLeg)
+                .append(" and the second leg is ")
+                .append(secondLeg)
+                .append(" the color is: ")
+                .append(color);
+        System.out.println(builder.toString());
```

Comment:
Directly using `System.out.println` in the `draw()` method makes it less flexible and harder to test. Consider returning the string from `draw()` and handling the printing elsewhere, or using a logging framework instead.